packages/auth/amplify_auth_cognito/pigeons/native_auth_plugin.dart

```diff
@@ -45,8 +45,33 @@ abstract class NativeAuthPlugin {
 
 @HostApi()
 abstract class NativeAuthBridge {
+  /// Configure the native platform/plugin.
+  ///
+  /// On iOS/Android, this calls `Amplify.addPlugin` with the [NativeAuthPlugin]
+  /// implementation.
   @async
   void configure();
+
+  /// Sign out by presenting [url] and waiting for a response to a URL with
+  /// [callbackUrlScheme].
+  ///
+  /// If [preferPrivateSession] is `true`, do not persist session cookies.
+  @async
+  Map<String, String> signIn(
```

Comment:
*nit:* `signInWithUrl` would make it clear that this has nothing to do with username/password auth.